DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core.UnitTests/TestConfigurations/configuration-miss-rules.json

```diff
@@ -1,19 +1,19 @@
 ﻿{
     "defaultSettings": {
         "perturb": {
-            "span": "1",
+            "span": 1,
             "roundTo": 2,
             "rangeType": "Proportional",
             "distribution": "Uniform"
         },
         "dateShift": {
             "dateShiftKey": "123",
             "dateShiftScope": "SeriesInstance",
-            "dateShiftRange": "50"
-        },   
+            "dateShiftRange": 50
+        },
         "cryptoHash": {
             "cryptoHashKey": "123",
-            "cryptoHashFunction": "sha256"
+            "cryptoHashFunction": "hmacsha256"
```

Comment:
Looks like not all settings are used for now.
hash function even doesn't appear in CryptoHashSetting class.
If currently only one specific hash function is supported, it's a little confusing that allow user to set this field but don't use it. We can add this setting later when more functions are supported.

Reply:
We removed it.